database/src/main/resources/db/V2__Tag_System.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE tags
```

Comment:
Should the file name be representative of the action eg. V2__Create_table_tags.sql? That way looking at a resource folder would give a clear idea on what each of the migrations represent.
Just a thought

Reply:
The other file is also named like that

Reply:
May be rename both the files instead.